externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/udf/KDFRegistry.scala

```diff
@@ -88,5 +106,318 @@ object KDFRegistry {
     for (func <- registeredFunctions) {
       spark.udf.register(func.name, func.udf)
     }
+
+    if (extraUdfsNames.length != 0 && extraUdfsClasses.length != 0 &&
```

Comment:
we can get these from `spark.conf`

Reply:
I don't get what you mean can you explain more please ?

Reply:
replace `SparkEnv.get.conf ` with `SparkSession.conf`